### ISSUE TITLE
Create blog specific hook

### ISSUE THREAD

**Issue by selwynorren** [thirtybees/beesblog#20]
Howdy,

Is there a possibility of creating a specific hook for the Blog so that we can plugin in the side column modules for the blog to only show on the blog and not on the other pages. Something like
displayBeestBlogLeft

That way we can position the recent posts, categories and popular posts into this hook?

**Comment by Dh42** [comment]
This can actually be done by limiting the modules display in the positions page. Making new hooks just for the blog page would require us to add those new specific hooks to every module. Using the positions and limiting the page display is going to be the best possible way to do this.

**Comment by selwynorren** [comment]
Hi,
That would unfortunately not solve the issue. Which ever way you limit the modules, it will limit it on all pages not just blog specific pages. What I am hoping to achieve is only show specific modules for pages that contain blog content only. If I limit for instance latest product, it will affect it everywhere not just on blog pages. Or do I misunderstand?

**Comment by Dh42** [comment]
Can you give me a an idea of what you mean, like show this module on this type of page and this one on this other type of page.

**Comment by selwynorren** [comment]
Lesley, sure, well I can only really demonstrate half of the idea, as I have no idea how to create a custom hook in Thirty Bees (I really need to learn how to do more stuff - lol)

This is my core blog page:
https://www.orren-life.com/journal

On the left we have the categories, information, manufacturers and my account information. All sidebar modules that I really want to have on normal pages. On this page and even blog detail pages, all I want on there is the blog specific sidebar modules, these would be the journal/blog categories and recent entries.

With a blog specific hook, I can allocate modules in and out of there and change up teh mix far easier, then if I had to specify exceptions for each module.

I hope I answered your question
